src/coreclr/inc/clrnt.h

```diff
@@ -501,4 +501,47 @@ RtlVirtualUnwind(
 
 #endif // TARGET_RISCV64
 
+#ifdef TARGET_WASM
+//
+// Define unwind information flags.
+//
+
+#define UNW_FLAG_NHANDLER               0x0             /* any handler */
+#define UNW_FLAG_EHANDLER               0x1             /* filter handler */
+#define UNW_FLAG_UHANDLER               0x2             /* unwind handler */
+
+PEXCEPTION_ROUTINE
+RtlVirtualUnwind (
```

Comment:
This should not be needed for WASM. This stuff is windows only.

Reply:
This is used in
```
src/coreclr/vm/stackwalk.cpp:516:9: error: use of undeclared identifier 'RtlVirtualUnwind'
    516 |         RtlVirtualUnwind(0,
        |         ^
  1 error generated.
```
and is inside chunk of code with `FEATURE_EH_FUNCLETS` ifdef.

What do you think about this feature for wasm at this point? Should I keep it enabled?

Reply:
Yes, you want to keep `FEATURE_EH_FUNCLETS` enabled.